OpenEdXMobile/src/main/java/org/edx/mobile/event/MoveToDiscoveryTabEvent.kt

```diff
@@ -0,0 +1,19 @@
+package org.edx.mobile.event
+
+import android.os.Bundle
+import org.edx.mobile.deeplink.ScreenDef
+import org.edx.mobile.view.Router
+
+class MoveToDiscoveryTabEvent(@ScreenDef screenName: String? = null) {
+
+    var screenArgumentsEvent: ScreenArgumentsEvent? = null
+        private set
+
+    init {
+        if (screenName != null) {
```

Comment:
Conceptually we shouldn't create another event within this event. It shouldn't be the responsibility of this class.

We can create an instance creation function within ScreenArgumentsEvent class and pass required params in it. like 

```
    public static ScreenArgumentsEvent getEventInstance(params) {

    }

```